data/systemd/snapd-user.service.in

```diff
@@ -0,0 +1,10 @@
+[Unit]
```

Comment:
I bet I'm missing something but isn't it the case that user services have this funky syntax including `@` in the service name?